libfwupd/fwupd-error.h

```diff
@@ -30,6 +30,7 @@ G_BEGIN_DECLS
  * @FWUPD_ERROR_PERMISSION_DENIED:		Permission was denied
  * @FWUPD_ERROR_BROKEN_SYSTEM:			User has configured their system in a broken way
  * @FWUPD_ERROR_BATTERY_LEVEL_TOO_LOW:		The system battery level is too low
+ * @FWUPD_ERROR_NEEDS_USER:			User needs to do an action to complete the update
```

Comment:
`NEEDS_USER` doesn't mean much. `NEEDS_USER_ACTION` might be better, surely?

Reply:
Will adjust